src/main/java/frc/robot/Robot.java

```diff
@@ -45,5 +46,24 @@ public void teleopPeriodic() {
     } else if (arcadeDrive == false) {
       m_tankDrive.tankDrive(m_controller.getLeftY(), -m_controller.getRightY(), true);
     }
+
+    // X-button turns motor on forward
+    if (m_controller.getXButtonReleased())
+    {
+      m_armWinchMotor.set(0);
+    }
+    else if (m_controller.getXButtonPressed()) {
+      m_armWinchMotor.set(0.1);
```

Comment:
10% will probably be too low to move the motor, but we can adjust when we get it wired up tomorrow

Reply:
agreed